new-community-project.html

```diff
@@ -0,0 +1,50 @@
+---
+layout: fullwidth
+primary_title: 'New Community Project'
+---
+<p>
+    Adding a new community project is accomplished through a pull request to <a href="https://github.com/opensearch-project/project-website">opensearch-project/project-website</a> in the <code>_community_projects</code> folder. The suggested method is to fork the repo, add the <a href="https://github.com/opensearch-project/project-website/blob/main/_community_projects/_sample.md">community project markdown file</a> and <a href="https://github.com/opensearch-project/project-website#building-the-website">build the website locally to verify it looks correct</a> then contribute the changed file back to opensearch-project/project-website.
```

Comment:
I believe it would make sense to have the three links open in a new tab - this allows the user the remain on the 'new community project page' but open the examples/samples as reference.

Reply:
Fixed!